File: java/Product.java
class Product 
{
    protected int idProduct;
    protected String name;
    protected String brand;
    protected String price;

    public Product(int idProduct, String name, String brand, String price) 
    {
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public String toTableRow() 
    {
        return String.format("| %-7d | %-15s | %-10s | %-12s |", idProduct, name, brand, price);
    }
}

File: php/Product.php
<?php

class Product
{
    protected $idProduct;
    protected $name;
    protected $brand;
    protected $price;

    public function __construct($idProduct, $name, $brand, $price)
    {
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    public function getIdProduct()
    {
        return $this->idProduct;
    }

    public function getName()
    {
        return $this->name;
    }

    public function getBrand()
    {
        return $this->brand;
    }

    public function getPrice()
    {
        return $this->price;
    }
}

?>